app/src/main/java/io/gnosis/safe/ui/transactions/TransactionView.kt

```diff
@@ -154,7 +154,7 @@ sealed class TransactionView(
         val transactionHash: String
     )
 
-    data class SectionHeader(@StringRes val title: Int, override val id: String = "<unused>") : TransactionView(TransactionStatus.PENDING, id)
+    data class SectionHeader(val title: String, override val id: String = "<unused>") : TransactionView(TransactionStatus.PENDING, id)
```

Comment:
we should use localization on the client-side. I suggest ignoring the label field and continue using local string resources. You could communicate to client-gw that the label ist not needed.

Reply:
The date (long) needs to be localized and "timezoned". But is that part of this PR of data loading? SectionHeader should probably receive a date instance and not a string here.

Reply:
Isn't `SectionHeader` for `Next` and `Queued`? Either way, maybe it would be easier to store the `@StringRes Int`. As for dates, I agree with Dirk as in, either providing the `viewHolder` with the date formatter or format the date in the `viewModel` where the list is created. I would recommend having an additional type here `DateHeader` or something like that, then you don't have to coerce everything into a `String`

Reply:
Labels are supposed to tell us what the `next` tx is. Don't we need that information? We should not show these Lables to the user. But we need to know which is the next tx, don't we?

Reply:
Yes we do. I wouldn't get rid of them. Actually they are part of the mocks AFAIK: https://projects.invisionapp.com/share/W4ZBLJ2SHMB#/screens

The `Label` coming from the backend is just a marker within the list for `next` and `queued` sections, only returned in the `/transactions/queued` endpoint. The clients are expected to handle localisation. 

That's why my suggestion was to use `@StringRes Int` . But I am not sure specifically for what this `TransactionView` subclass will be used for. The backend returns different types for `Label`, `DateLabel` and `ConflictHeader` . I wouldn't recommend using a single `TransactionView` subclass for all 3 types, but that's up to you.